src/argilla/client/feedback/dataset.py

```diff
@@ -527,7 +560,19 @@ def delete_and_raise_exception(dataset_id: str, exception: Exception) -> None:
 
             for question in self.questions:
                 try:
-                    datasets_api_v1.add_question(client=httpx_client, id=argilla_id, question=question.dict())
+                    datasets_api_v1.add_question(
+                        client=httpx_client,
+                        id=argilla_id,
+                        question=question.dict(
+                            include={
+                                "name": True,
+                                "title": True,
+                                "description": True,
+                                "required": True,
+                                "settings": True,
+                            }
+                        ),
```

Comment:
I think you can use directly a set in this case:

```suggestion
                        question=question.dict(
                            include={ "name", "title", "description", "required", "settings" }
                        ),
```